padevents/padevents.py

```diff
@@ -886,6 +890,10 @@ async def do_partial(self, ctx, server):
         # active_events = list(group_to_active_event.values())
         # pending_events = list(group_to_pending_event.values())
 
+        if group != None:
```

Comment:
I would put these a few lines above so that it is right after the original declaration of active_events and pending_events.  This is also another case where you should use `is not` instead of `!=`